src/Stratis.Bitcoin.Features.PoA/Voting/IWhitelistedHashesRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.PoA.Voting
+{
+    public interface IWhitelistedHashesRepository
```

Comment:
No need to add it in a separated file, we do not expect more than 1 implementation of that interface so it can be above the implementation, in the same file

Reply:
I needed the abstraction for testing.